### PR TITLE
Address feedback about existing code from 5248

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: service/mapresolver.go [hunk 1]
```diff
@@ -94,23 +97,23 @@ func (mr *mapResolver) Resolve(ctx context.Context) (*config.Map, error) {
...
+		if idx := strings.Index(uri, ":"); idx != -1 && !driverLetterRegexp.MatchString(uri) {
+			scheme = uri[:idx]
 		} else {
-			location = scheme + ":" + location
+			uri = scheme + ":" + uri
 		}
 		p, ok := mr.mapProviders[scheme]
 		if !ok {
-			return nil, fmt.Errorf("scheme %v is not supported for location %v", scheme, location)
+			return nil, fmt.Errorf("no provider found for the scheme=%v, when resolving the uri=%v", scheme, uri)
```
Comment (jpkrohling) [inline service/mapresolver.go:114]: Very minor, but I prefer the older version, with `%q` instead of `%v`. IMO, it's more readable.

**Review by jpkrohling — APPROVED** [review]
Just a minor comment, but I guess it's a matter of taste. So, LGTM.

File: service/mapresolver.go (hunk 1 above)
Comment (bogdandrutu) [reply service/mapresolver.go]: Done.

Comment (bogdandrutu) [reply service/mapresolver.go]: You want to change to the old message?

**Pull request merged**